CDCgov/prime-reportstream
# 15154: document KQL example queries for looking at the events

This PR adds some example queries for looking at the new events and fixes an issue with the digest extraction

Test Steps:

Testing the queries
1. Go to app insights in test or staging and try the queries. (I know staging had some data that was not routed for DC, but you can also try just sending reports to the ignore org yourself)

Confirming the update extraction
1. Run the smoketest
2. Watch the logs for ITEM_ACCEPTED, ITEM_FILTER_FAILED events and make sure you see the orderingFacility state populated as well as the tracking ID in the logged event 

## Changes
- Updates azure-events.md with additional examples for querying the new reportstream events
- Updates the bundle digest instances in FHIR engine to take a CustomContext so that `resolve` works correctly in a FHIR path

## Checklist

### Testing

### Process

## Linked Issues
- #15154 

## To Be Done


## Specific Security-related subjects a reviewer should pay specific attention to
